src/App.jsx

```diff
@@ -24,17 +26,39 @@ import { differenceInDays } from 'date-fns';
 function App() {
   const [answer, setAnswer] = useState();
   const [volume, setVolume] = useState(2);
+  const [isMidnight, setIsMidnight] = useState(false);
   const [mobileOrSafari, setMobileOrSafari] = useState(false);
+  const [testMode, setTestMode] = useState(false);
+  const [showStats, setShowStats] = useState(false);
 
   function setGameIndex() {
-    let startDate = new Date('2023-05-11');
+    let startDate = new Date('2023-05-15');
     // Find number of days between now and startDate
     return differenceInDays(new Date(), startDate)-1 > 0 ? differenceInDays(new Date(), startDate)-1 : 0;
   }
 
+  function clearStorage() {
+    window.localStorage.removeItem('perfectPitchPuzzleStats');
+  }
+
+  
+
   useEffect(() => {
-    let randomIndex = Math.floor(Math.random() * data.length)
-    setAnswer(data[setGameIndex()]);
+    
+    const urlSearchParams = new URLSearchParams(window.location.search);
+    const searchParamsObject = {};
+    // Iterate over each search parameter and store the values in the object
+    for (const [key, value] of urlSearchParams) {
+      searchParamsObject[key] = value;
+    }
+    if (searchParamsObject.testMode === 'true') {
```

Comment:
Might want to add comment about how to enter testmode, e.g. append `?testMode=true` to URL